qtglean/ios.cmake

```diff
@@ -96,15 +96,29 @@ target_sources(iosglean PUBLIC
     ${CMAKE_CURRENT_BINARY_DIR}/glean/gleanFFI.h
 )
 
+list(APPEND PINGS_LIST ${CMAKE_SOURCE_DIR}/src/shared/telemetry/pings_deprecated.yaml)
+list(APPEND METRICS_LIST ${CMAKE_SOURCE_DIR}/src/shared/telemetry/metrics_deprecated.yaml)
+
+get_filename_component(PROJECTS_DIR ${CMAKE_SOURCE_DIR}/src/apps ABSOLUTE)
+file(GLOB PROJECTS_NAMES LIST_DIRECTORIES true RELATIVE ${PROJECTS_DIR} ${PROJECTS_DIR}/*)
+foreach(PROJECT ${PROJECTS_NAMES})
+    if(EXISTS ${PROJECTS_DIR}/${PROJECT}/telemetry/pings_deprecated.yaml)
+        list(APPEND PINGS_LIST ${PROJECTS_DIR}/${PROJECT}/telemetry/pings_deprecated.yaml)
+    endif()
+    if(EXISTS ${PROJECTS_DIR}/${PROJECT}/telemetry/metrics_deprecated.yaml)
+        list(APPEND METRICS_LIST ${PROJECTS_DIR}/${PROJECT}/telemetry/metrics_deprecated.yaml)
+    endif()
+endforeach()
```

Comment:
I think we are missing the addition of the non-deprecated metrics files as well.